.github/workflows/ci.yaml

```diff
@@ -37,23 +44,19 @@ jobs:
           python-version: ${{ matrix.python }}
           cache: 'pip'
 
-      - uses: actions/cache@v3
-        with:
-          path: ~/.cache/pip
-          key: ${{ runner.os }}-${{ matrix.python-version }}-pip-${{ hashFiles('minimum-requirements-ci.txt', 'pyproject.toml') }}
-          restore-keys: |
-            ${{ runner.os }}-${{ matrix.python-version }}-pip-
-
       - name: Install required Python packages
         run: |
           python -m pip install --upgrade pip
-          python -m pip install nox toml
+          python -m pip install -e .[dev-noxfile]
 
-      - name: run nox
+      - name: Run nox
+        # To speed things up a bit we use the speciall ci_checks_max session
```

Comment:
typo in `speciall`

Reply:
Woops, but this needs to be fixed in `repo-config`. I think it is fine to introduce the typo here as it is an auto-generated file, but I will fix in `repo-config`.

Reply:
https://github.com/frequenz-floss/frequenz-repo-config-python/pull/81